README.md

```diff
@@ -0,0 +1,145 @@
+            0x18, 
+    }
+
+    keyTriples, err := store.GetKeyTriples(&env, "cca", false)
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fmt.Printf("key: %s\n", string(keyTriples[0].KeyList[0].KeyBytes))
+}
+```
+
+## CLI
+
+This repo includes a CLI tool, `corim-store`, for interacting with the store.
+Use `-h`/`--help` option to view available commands and their arguments.
+
+### Example flow
+
+```bash
+./corim-store db init
+```
+Initialize the database for use with the store.
+
+```bash
+./corim-store db status
+```
+Show the database connection settings and current migration status. This can be
+used to make sure that the database is up-to-date with what the store expects.
+This should be run after updating to the latest version of the store. If new
+migrations are available, the database can be updated by re-running the
+`mirgrate` (but not `init`!) command.
+
+```bash
+./corim-store corim add sample/corim/*.cbor
+```
+Add sample CoRIM's to the store.
+
+```bash
+./corim-store list module-tags
+```
+List module tags (CoMID's) inside the store.
+
+```bash
+./corim-store get --class-id 2QJYWCB/RUxGAgEBAAAAAAAAAAAAAwA+AAEAAABQWAAAAAAAAA
+```
+Get endorsements associated with the specified class ID from the store.
+
+```bash
+./corim-store corim dump cca-ref-plat -o /tmp/cca-platform-ref-vals.cbor
+```
+Write the contents associated with the specified manifest ID (here,
+`cca-ref-plat`) to file as an unsigned CoRIM.
+
+### Configuration
+
+`corim-store` accepts configuration in YAML format. By default, configuration
+is read from `~/.config/corim-store.yaml` if it exists. An alternative file
+can be specified with `--config` command line option.
+
+The following settings are accepted:
+
+- `dbms`: The DBMS backing the store. Accepted values are `sqlite`, `postgres`,
+  and `mysql` (TODO: currently, only `sqlite` is supported). The default is
+  `sqlite`.
+- `dsn`: The Data Source Name. This is the string used to connect to the
+  database. The format depends on the DBMS (search the DBMS documentation for
+  "data source name" or "connection string"). The default is
+  `file:store:db?cache=shared`.
+- `require-label`: A boolean value indicating whether a label MUST be specified
+  for added endorsements. The default is `false`. (Note: without this flag, 
+  labels may still be specified; it is just that they are not _required_).
+  Labels allow partitioning contained endorsements under different "namespaces"
+  indicated by the label.
+-  `hash-alg`: hashing algorithm used to calculated store-internal digests.
+   (Note: this has no impact on the digests included inside
+   endorsements/reference values). Accepted values are `md5`, `sha256`, and
+   `sha512`. The default is `sha256`.
+- `trace-sql`: A boolean value indicating whether to log executed SQL
+  statements to STDERR. The default is `false`.
+- `insecure`: A boolean value indicating whether insecure transactions (e.g.
```

Comment:
Could you please also clarify, whether UnsignedCoRIMs can also be allowed or not, submitting to the stores??